CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 1.32.5
+
+* Use an updated version of `node_preamble` when compiling to JS.
```

Comment:
Generally, we try to keep changelog entries focused on the user's perspective. What does the latest version of `node_preamble` allow/fix that older versions did not?